cloud/disk_manager/internal/pkg/dataplane/url/vhd/common.go

```diff
@@ -0,0 +1,29 @@
+package vhd
+
+import "github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/dataplane/url/common"
+
+////////////////////////////////////////////////////////////////////////////////
+
+type imageMapEntry struct {
```

Comment:
This structure duplicates the common.ImageMapItem, I do not think it is required.

Reply:
it is not a duplicate. i need it to implement mergeable and dumpToItem methods

code becomes easier 

we have the same structures in other formats:
https://github.com/ydb-platform/nbs/blob/bf37ab3e6fb00f5093ccd4d589f9875edd0e0abf/cloud/disk_manager/internal/pkg/dataplane/url/qcow2/common.go#L90

Reply:
I will look into the merge mechanism further.